dev/django.Dockerfile

```diff
@@ -22,8 +22,7 @@ RUN pip install -U pip && \
     pip install --editable /opt/django-project[dev] && \
     pip install --editable /opt/django-project/swcc
 
-RUN export url=$(curl -s https://api.github.com/repos/SCIInstitute/ShapeWorks/releases | grep -o "http.*dev-linux.*${6:-tar.gz}"); \
-    curl -L -o /tmp/shapeworks.tgz $url
+RUN curl -L -o /tmp/shapeworks.tgz https://api.github.com/repos/SCIInstitute/ShapeWorks/tarball/dev-linux
```

Comment:
If we combine the RUN command with curl with the RUN command that untars the file, the docker image will end up smaller by the size of the tarball.

Reply:
Ok, changed in [2b011bf](https://github.com/girder/shapeworks-cloud/pull/214/commits/2b011bf7b674f83356eca5ffa19f00cfaeff72f0)